### PR TITLE
add model to clean speech with low latency (delay<40ms)

### PR DESCRIPTION
**Pull request by krodyush**
This is DNS model to suppress noise and keep speech with low delay (<40ms) and small processing patch (128 samples).

### CODE REVIEW

File: tools/accuracy_checker/configs/noise-suppression-denseunet-ll-0001.xml [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../models/intel/noise-suppression-denseunet-ll-0001/accuracy-check.yml
```
Comment (vladimir-dudnik) [inline tools/accuracy_checker/configs/noise-suppression-denseunet-ll-0001.xml:1]: @krodyush please rename soft link from configs/noise-suppression-denseunet-11-0001.xml to yml

**Review by vladimir-dudnik — CHANGES_REQUESTED** [review]

File: tools/accuracy_checker/configs/noise-suppression-denseunet-ll-0001.xml (hunk 1 above)
Comment (krodyush) [reply tools/accuracy_checker/configs/noise-suppression-denseunet-ll-0001.xml:1]: Thanks!

**Review by eaidova — APPROVED** [review]

**Comment by eaidova** [comment]
@vladimir-dudnik  in internal repo request is merged, could you please check this PR again?

File: models/intel/noise-suppression-denseunet-ll-0001/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+
+## Input
+
+Sequence patch, name: `input`, shape: `1, 128`, format: `B, T`, where:
+
+ - `B` - batch size
+ - `T` - number of samples in patch
+
+input states, names: `inp_state_*`, should be filled by corresponding `out_state_*` from previous step
```
Comment (vladimir-dudnik) [inline models/intel/noise-suppression-denseunet-ll-0001/README.md:45]: would not it be useful to state nubmer of inp_state_* inputs, just for the clarity?

File: models/intel/noise-suppression-denseunet-ll-0001/README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## Output
+
+Sequence patch, name: `output`, shape: `1, 128`, format: `B, T`, where:
+
+ - `B` - batch size
+ - `T` - number of samples in patch
+Note: The output patch is "shifted" by 384 (24ms) samples in time. So output[0,i] sample is synced with input[0,i-384] sample
+
+output states, names: `out_state_*`, should be used to fill corresponding `inp_state_*` on next step
```
Comment (vladimir-dudnik) [inline models/intel/noise-suppression-denseunet-ll-0001/README.md:55]: same about out_state_* outputs, would be good to know their number

**Review by vladimir-dudnik — APPROVED** [review]
looks good, although, please look at comments about specifying number of inputs/outputs, that would be good to know exact number from model description, rather then calculating them in debugger.

File: models/intel/noise-suppression-denseunet-ll-0001/README.md (hunk 3 above)
Comment (krodyush) [reply models/intel/noise-suppression-denseunet-ll-0001/README.md]: ok

File: models/intel/noise-suppression-denseunet-ll-0001/README.md (hunk 2 above)
Comment (krodyush) [reply models/intel/noise-suppression-denseunet-ll-0001/README.md]: ok

**Pull request merged**